packages/devtools/tool/analysis_options_check.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Check that devtools/analysis_options.yaml and
+/// devtools_server/analysis_options.yaml are identical.
+
+import 'dart:io';
+
+void main() {
+  final File devToolsOptions = File('../devtools/analysis_options.yaml');
+  if (!devToolsOptions.existsSync()) {
+    fail('${devToolsOptions.path} not found.');
+  }
+
+  final File devToolsServerOptions =
+      new File('../devtools_server/analysis_options.yaml');
+  if (!devToolsServerOptions.existsSync()) {
+    fail('${devToolsServerOptions.path} not found.');
+  }
+
+  final devToolsOptionsContent = devToolsOptions.readAsLinesSync();
+  final devToolsServerOptionsContent = devToolsServerOptions.readAsLinesSync();
+
+  for (int i = 1; i < devToolsOptionsContent.length; i++) {
```

Comment:
comment on why you are skipping the first line. Otherwise it looks like an off by 1 error.

Reply:
added comment.